File: SumOfArray.py
from random import randint

def sum(arr, size):
    if size == 0:
        return arr[size]
    else:
        return arr[size] + sum(arr,size-1)

if __name__ == "__main__":
    n = int(input("Enter Size Of Array : "))
    arr = []
    for i in range(n):
        arr.append(randint(0,100))
    for i in arr:
        print(i,end=" ")
    print()
    print("Sum Of Array is ",sum(arr,n-1))

File: SumOfArray.cpp
#include <bits/stdc++.h>
using namespace std;

int sum(int arr[], int size)
{
  if (size == 0)
    return arr[size];
  else
    return arr[size] + sum(arr, size-1);
}

int main(int argc, char const *argv[])
{
  cout << "Enter size of Array : ";
  int size;
  cin >> size;
  int arr[size];
  srand(time(0));
  for(int i=0; i<size; i++)
    arr[i] = rand()%100;
  for(int i=0; i<size; i++)
    cout << arr[i] << " ";
  cout << endl;
  cout << "\nSum of Array is : " << sum(arr, size-1) << endl;
  return 0;
}
